frontend/src/app/components/work-packages/wp-relations-count/wp-watchers-count.component.ts

```diff
@@ -1,6 +1,8 @@
 import {Component, Input, OnInit} from '@angular/core';
 import {WorkPackageCacheService} from '../../work-packages/work-package-cache.service';
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
+import { WorkPackageWatchersService } from 'core-app/components/wp-single-view-tabs/watchers-tab/wp-watchers.service';
+import { HalResource } from 'core-app/modules/hal/resources/hal-resource';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/b7fbac0a11282658bf32cf3a5791f6592828840c/frontend/src/app/components/work-packages/wp-relations-count/wp-watchers-count.component.ts#L6"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>